### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by msabramo**
Adds a `Dockerfile` with a simple Kotti server.

```
❯ docker build -t msabramo/kotti .
...
❯ docker run -i -t -P msabramo/kotti
Starting server in PID 1.
serving on http://0.0.0.0:5000
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: python
+sudo: false
```
Comment (disko) [inline .travis.yml]: Please revert, it's already in #377.

**Comment by disko** [comment]
I haven't used or even tried Docker yet.  What exactly would we gain by adding this?

**Comment by msabramo** [comment]
You could use the Dockerfile to build a docker image and then upload it to the docker registry as say, kotti/kotti. 

Now someone who wants to try Kotti can do `docker pull kotti/kotti` to download the image and then they can run a container with it. 

The user did not need to do virtualenv or pip install or to even have Python installed. 

Let me know if you have more questions. You'd need to build and upload the Docker image and then we would add docs telling users how to download and run it.

File: .travis.yml (hunk 1 above)
Comment (msabramo) [inline .travis.yml]: Done.

**Comment by disko** [comment]
Thanks for the explanation.  I still want to try and fully understand it myself before merging, unless @dnouri already does and clicks the button instantaneously.

**Comment by dnouri** [comment]
I know Docker's the go-to deployment format for a lot of people these days.  I think it's a good idea to maintain the Dockerfile within Kotti.  Can we put a few sentences in the docs for this?

**Comment by msabramo** [comment]
Absolutely. Want to log on to the Docker Registry and upload a kotti/kotti or kotti/demo image? Then I can refer to it in the docs that I write.

**Comment by disko** [comment]
> Want to log on to the Docker Registry and upload a kotti/kotti or kotti/demo image?

Yes we want.  I've created an account over there and claimed the kotti organization.

> Then I can refer to it in the docs that I write.

Please do so. Thanks in advance!

**Comment by msabramo** [comment]
Cool, I see that you created the kotti organization on the Docker Hub.

What I'd recommend is that you set up an "automated build" on the Docker hub (this way the Docker servers build the image for you automatically from the Dockerfile and you don't even have to have Docker installed or know how to use it):

0. Merge this PR to master. If you're opposed to merging this to master before I add the docs, you could skip this and have the automated build pull from a branch, but then you'd need to change it later after merging, so I think it might be easier to just merge this change to master now.
1. Go to https://registry.hub.docker.com/builds/github/select/
2. Select the "Kotti" GitHub repo
3. Set up an automated build for namespace "kotti" and (Docker) repository "kotti". You can keep the "Tags" section defaults of pulling from the master branch unless you didn't merge to master in step 0, in which case this will have to be tweaked.
4. Click "Create Repository" button.
5. Let me know whether it works or not and then I can document how an end-user can pull the image and create/run a Kotti container from it.

**Comment by disko** [comment]
Done as described: https://registry.hub.docker.com/u/kotti/kotti/.  *Seems* to work, it was built successfully, but I didn't test the result.

**Comment by msabramo** [comment]
I tested the result. It works great. Thanks!

I'll open a PR soon to add instructions to the README.

**Pull request merged**